src/components/projectDetail/AddressAddModal.tsx

```
@@ -0,0 +1,110 @@
+    address: string
+    detailAddress: string
+    zipcode: string
+  }
+  setNewAddress: (addr: { address: string; detailAddress: string; zipcode: string }) => void
+  isAddressModalOpen: boolean
+  setIsAddressModalOpen: (open: boolean) => void
+  handleInputChange: (
+    e: React.ChangeEvent<HTMLInputElement>,
+    field: "address" | "detailAddress" | "zipcode"
+  ) => void
+  handleComplete: (data: any) => void
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)